utils/go-yq/wrapper.py

```
@@ -0,0 +1,77 @@
+        "ini": "ini",
+        "xml": "xml",
+    }
+
+    # Ensure user did not provide any format information
+    if not (is_arg(long_option, extra) or is_arg(short_option, extra)):
+        # Check unique format in input and output separately
+        if len(set(formats)) != 1:
+            raise ValueError(
+                f"The following files: {files}, should have the same extension. "
+                f"Got {set(formats)} instead."
+            )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)